src/Main.elm

```diff
@@ -147,8 +148,15 @@ type alias FormField =
     }
 
 
+type alias ShortTextAttrs =
+    { type_ : String
+    , maxlength : Maybe Int
+    , multiple : Maybe Bool
+    }
+
+
 type InputField
-    = ShortText String (Maybe Int)
+    = ShortText ShortTextAttrs
```

Comment:
[yes I said:](https://github.com/choonkeat/tiny-form-fields/pull/2#pullrequestreview-2260711483)

> feels like we need to provide a way to indicate extra attributes, instead of appending more and more `(Maybe a)`]

But making it a [Record](https://elm-lang.org/docs/records) and adding more and more fields isn't much better either.

I'm thinking we might need a `List (String, String)` attributes. especially if these attributes are for browser behaviour, not something we rely on for logic in tff code